examples/exempt_custom_fees/main.go

```diff
@@ -166,6 +142,27 @@ func main() {
 	 *
 	 * Collector 0.0.B sends 10_000 units of the token to 0.0.A.
 	 */
+
+	// First we transfer the amount from treasury account to second account
+	treasuryTokenTransderTransaction, err := hedera.NewTransferTransaction().
+		AddTokenTransfer(tokenId, id, -10000).AddTokenTransfer(tokenId, secondAccountId, 10000).
+		FreezeWith(client)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Something I noticed now is that in this example in a lot of cases we only print the error without returning. Perhaps we should we add `return`s in all those cases?